lnwire/fuzz_test.go

```diff
@@ -807,3 +807,17 @@ func FuzzUpdateFulfillHTLC(f *testing.F) {
 		harness(t, data)
 	})
 }
+
+func FuzzCustomMessage(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte, customMessageType uint16) {
+		if customMessageType < uint16(CustomTypeStart) {
```

Comment:
In case the type is below the custom type, should we just add `CustomTypeStart` to it? Otherwise in 50% of the cases the fuzzer won't even reach the target code.

Reply:
yes good point, added it